### ISSUE TITLE
Grantlee is documented to be optional, but cmake errors out without it

### ISSUE THREAD

**Issue by Moini** [dragotin/kraft#123]
As the AppImage said the template file that was included doesn't work (see screenshot - weasyprint installed via pip3, not sure whether it's actually included or needs to be available on the system...?), I've tried to compile kraft.
![grafik](https://user-images.githubusercontent.com/3240233/141328181-f5e9b562-f0b8-4ab3-bf30-026c3d4c0f88.png)

Now I'm stuck at a contradiction between docs (https://github.com/dragotin/kraft/blob/41e10028cce5d9c3a9009687dbb68cab180b8c9d/INSTALL#L41) and cmake - is Grantlee required or not?

![grafik](https://user-images.githubusercontent.com/3240233/141329129-b5b1d92d-dee9-44ef-afdb-1f98d9c8265f.png)

Some package names for installation on Ubuntu:

extra-cmake-modules  
libkf5i18n-dev  
libkf5xmlgui-dev  
libkf5contacts-dev

Q1: Is it expected that the AppImage html-type template does not work?  
Q2: Is weasyprint included or does it need to be installed separately?  
Q3: Is the documentation outdated or is the cmake script too strict?

**Comment by dragotin** [comment]
Thanks for this report.... Nasty stuff and hard to check on a developer machine where everything is installed.

I updated the INSTALL file, hopefully it is more accurate now. Btw, libkf5xmlgui-dev should not be needed any more, I cleaned the KF5 deps at some point.

Indeed grantlee is mandatory, since it is way better than the ctemplate solution keeping it obsolete only lasted shortly ;-)

Answers to your questions:
- Q1: Yes, still. The AppImage needs more work as you proofed now. The available AppImage does not have the weasyprint installed yet.
- Q2: It is not in the AppImage yet. If you install it in the host system, there is a good chance that it is picked up by the AppImage as well. But it always has to be installed separately, it is not included in Kraft.
- Q3: Documentation was outdated. Fixed.
